app/controllers/user.js

```diff
@@ -0,0 +1,39 @@
+const { validationResult } = require('express-validator/check');
+const bcrypt = require('bcryptjs');
+const salt = bcrypt.genSaltSync(10);
+const { databaseError } = require('../errors');
+const { bad_request_error } = require('../errors');
+const { conflict_error } = require('../errors');
```

Comment:
This requires could be merged in one, `const { error1, error2, ... } = require('errors')`